oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/oidc/web/OidcProviderConfigurationEndpointFilterTests.java

```diff
@@ -71,6 +84,53 @@ public void doFilterWhenNotConfigurationRequestThenNotProcessed() throws Excepti
 
 		verify(filterChain).doFilter(any(HttpServletRequest.class), any(HttpServletResponse.class));
 	}
+	@Test
+	public void providerConfigurationParametersConverterWhenCustomThenAbleToOverride() throws Exception{
```

Comment:
Please rename method to be consistent with other methods...should start with `doFilter...`